src/Stratis.Features.FederatedPeg.IntegrationTests/NodeInitialisationTests.cs

```diff
@@ -43,7 +43,7 @@ public NodeInitialisationTests()
             this.scriptAndAddresses = FederatedPegTestHelper.GenerateScriptAndAddresses(this.mainNetwork, this.sidechainNetwork, 2, pubKeysByMnemonic);
         }
 
-        [Fact]
+        [Fact(Skip = "Broken. Re-enable when fixed")]
```

Comment:
Maybe Instead of commenting this out maybe just don't run this entire project tests

Reply:
Ok will update